### PR TITLE
Add docs page for statuses

### PR DESCRIPTION
**Pull request by srt32**
closes github/design-systems#1351

This change adds a basic page explaining how the statuses work

image

### CODE REVIEW

**Comment by joelhawksley** [comment]
@srt32 what do you think about including the criteria for the statuses we currently have in `CONTRIBUTING`?

**Comment by srt32** [comment]
Thoughts on fd6bee4?

File: docs/content/statuses.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Component status
+---
+
+Each component is given a status that explains the component's maturity level.
```
Comment (joelhawksley) [inline docs/content/statuses.md:5]:
```suggestion
Each component is assigned a status for its maturity level:
```

File: docs/content/statuses.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+title: Component status
```
Comment (joelhawksley) [inline docs/content/statuses.md:2]:
```suggestion
title: Statuses
```

File: docs/content/statuses.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| Status         | Meaning                                                                |
+| -------------- |------------------------------------------------------------------------|
+| `Alpha`        | Experimental. Breaking changes are likely.                             |
+| `Beta`         | Usage encouraged, but not required.                                    |
+| `Stable`       | Usage expected.                                                        |
+| `Deprecated`   | Usage not allowed. May be removed at any time.                         |
+
+We expect many components to move from alpha to beta and from beta to stable
+soon. Feel free to watch for releases in the
+[repo](https://github.com/primer/view_components).
```
Comment (joelhawksley) [inline docs/content/statuses.md:16]:
```suggestion
We expect many components to move from alpha to beta and from beta to stable soon. Feel free to watch for releases in the [repo](https://github.com/primer/view_components).
```

File: docs/content/statuses.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+You can also see component status at a glance on [this project
+board](https://github.com/primer/view_components/projects/3).
```
Comment (joelhawksley) [inline docs/content/statuses.md:19]:
```suggestion
You can also see component status at a glance on [this project board](https://github.com/primer/view_components/projects/3).
```

File: docs/content/statuses.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Criteria for changing component status
+
+### Promote from alpha to beta
+
+- The component does not use any deprecated `view_components` platform features (eg - Slots v1).
```
Comment (joelhawksley) [inline docs/content/statuses.md:25]:
```suggestion
- The component does not use any deprecated `view_component` framework features, such as Slots v1.
```

File: docs/src/@primer/gatsby-theme-doctocat/nav.yml [hunk 6]
```diff
@@ -5,6 +5,8 @@
       url: /contributing
     - title: System arguments
       url: /system-arguments
+    - title: Component status
```
Comment (joelhawksley) [inline docs/src/@primer/gatsby-theme-doctocat/nav.yml:8]:
```suggestion
    - title: Statuses
```

File: docs/content/statuses.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+- Documentation and storybook stories are complete.
+- The API is expected to be stable.
```
Comment (joelhawksley) [inline docs/content/statuses.md:27]: Did we want to add a criteria for a certain % or # of internal uses?

Comment (srt32) [reply docs/content/statuses.md:24]:
```suggestion
- The API is expected to be stable and the component is used in dotcom at least 3 times.
```

Something like this? I'm not sure we have a more specific but also broadly applicable at this point.

Comment (joelhawksley) [reply docs/content/statuses.md:24]: I think this is fine for now 👍🏻

**Comment by srt32** [comment]
@joelhawksley good for a merge here?

**Review by manuelpuyol — APPROVED** [review]

**Pull request merged**